it/uk/gov/hmrc/agentassurance/controllers/AmlsJourneyControllerISpec.scala

```diff
@@ -37,14 +37,15 @@ class AmlsJourneyControllerISpec extends PlaySpec
   val moduleWithOverrides: AbstractModule = new AbstractModule() {
     override def configure(): Unit = {
       bind(classOf[AmlsJourneyService]).toInstance(service)
+      bind(classOf[Clock]).toInstance(clock)
     }
   }
 
   protected def appBuilder: GuiceApplicationBuilder =
     new GuiceApplicationBuilder()
       .configure("microservice.services.auth.host" -> wireMockHost,
         "microservice.services.auth.port" -> wireMockPort,
-        "auditing.enabled" -> false
+        "auditing.enabled" -> false,
```

Comment:
I think the trailing comma will fail a jenkins build

Reply:
thx, fixed